index.js

```diff
@@ -141,7 +141,12 @@ module.exports = {
   },
 
   getParentAddonTree() {
-    return path.join(this.parentAddon.root, this.parentAddon.treePaths.src || this.parentAddon.treePaths.addon);
+    return path.join(
+      this.parentAddon.root,
+      this.parentAddon.modulePrefix ?
+        this.parentAddon.treePaths.addon :
+        this.parentAddon.treePaths.src
```

Comment:
Dunno, if this is the right solution, but apparently `modulePrefix` is missing in MU and I don't know a more direct way to detect MU.

Reply:
`modulePrefix` is missing initially, but _may_ be set indirectly by invoking other methods, so this is unsafe.

There will be an `isModuleUnfication` method on apps and addons, that checks for the presence of the `src` dir. Since it is not available yet, I've added a util that does the same.

Reply:
https://github.com/salsify/ember-css-modules/pull/128/commits/4e17d90e25d62d97bb3a0107a8886e256f354c07 and https://github.com/salsify/ember-css-modules/pull/128/commits/1d58c0ede351a1afa9b937cfc2ef4a804afc2b97